modules/apps/object/object-dynamic-data-mapping-storage/src/main/java/com/liferay/object/dynamic/data/mapping/storage/internal/ObjectDDMStorageAdapter.java

```diff
@@ -40,6 +63,22 @@ public DDMStorageAdapterDeleteResponse delete(
 			DDMStorageAdapterDeleteRequest ddmStorageAdapterDeleteRequest)
 		throws StorageException {
 
+		try {
+			ObjectEntry objectEntry = _objectEntryManager.getObjectEntry(
+				_getDTOConverterContext(null, null, null),
+				ddmStorageAdapterDeleteRequest.getPrimaryKey());
+
+			Map<String, Object> values = objectEntry.getProperties();
```

Comment:
@gabrielwas, shouldn't we name this variable as `properties` once we're filling it through the result of a method named `getProperties`?

Usually, this is a kind of consistency that Brian looks for. What do you think?

Reply:
You're right! Thanks, Renato! 😄